ihatemoney/forms.py

```diff
@@ -191,6 +197,8 @@ def fill(self, bill):
         self.external_link.data = bill.external_link
         self.date.data = bill.date
         self.payed_for.data = [int(ower.id) for ower in bill.owers]
+        if bill.type:
```

Comment:
What's the case where `bill.type` is set to `None`? (why do you need the `if` statement?)

Reply:
It can happens if you have some old bills stored in database. I wanted to avoid compatibility issues according to eneiluj.
> This parameter could be optional in the API (default to "loan") to avoid compatibility problems with old clients versions.

That's why I do a check here. Do you think we should add to the migration script a default value to `loan` for all previously saved bills ?

Reply:
Migrations ensure the database is valid for a specific schema, so we don't need to do defensive programing. Adding a default value of "loan" here seems the best way to handle this.

We probably want to do it on the model directly :-)